lib/publish.js

```diff
@@ -72,6 +73,7 @@ module.exports = async (pluginConfig, context) => {
     ...apiOptions,
     json: {
       /* eslint-disable camelcase */
+      name:name ? name : gitTag,
```

Comment:
Thanks for this - this matches GitLab's default behavior in version 12+.

**nitpick:** I personally like this phrasing better:

```suggestion
      name: name || gitTag
```